scripts/docker/build-docker-image.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+IMAGE_NAME=atomic-swap
+
+# VERSION can be "latest", a release tag, a hash or a branch name that does not contain slashes.
+# It must exist on github, local changes are not visible inside the container.
+VERSION=fde1cad9bf
```

Comment:
We'll change this to `master` or something before merging. I needed a commit from this branch that has the additional `swapd` environment variables, and I can't use the branch name as it has a `/` in it.

Reply:
It is set to master now in prep for merging.